src/status_im/contexts/preview/feature_flags/view.cljs

```diff
@@ -0,0 +1,39 @@
+(ns status-im.contexts.preview.feature-flags.view
+  (:require
+    [clojure.string :as string]
+    [quo.core :as quo]
+    [re-frame.core :as rf]
+    [react-native.core :as rn]
+    [status-im.contexts.preview.feature-flags.style :as style]
+    [status-im.feature-flags :as ff]))
+
+(defn view
+  []
+  [rn/view {:flex 1}
+   [quo/page-nav
+    {:type       :title
+     :text-align :left
+     :title      "Features Flags"
+     :icon-name  :i/arrow-left
+     :on-press   #(rf/dispatch [:navigate-back])}]
+   (doall
+    (for [context-name ff/feature-flags-categories]
+      (let [context-flags (filter (fn [[k]]
+                                    (string/includes? (str k) context-name))
+                                  (ff/feature-flags))]
```

Comment:
You can move this let binding to the `for` macro. Ref https://clojuredocs.org/clojure.core/for#example-542692c7c026201cdc326959